core/controllers/creator_dashboard.py

```diff
@@ -67,7 +67,7 @@ class NotificationsDashboardHandler(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
         job_queued_msec, recent_notifications = (
-            user_jobs_continuous.DashboardRecentUpdatesAggregator.get_recent_notifications(  # pylint: disable=line-too-long
+            user_jobs_continuous.DashboardRecentUpdatesAggregator.get_recent_user_changes(  # pylint: disable=line-too-long
```

Comment:
I believe you can break at the dot since the whole thing is in parens, in order to avoid the pragma. Want to try that?

Ditto below.

Reply:
Done.